frontend/src/graphql/mutations.js

```diff
@@ -233,6 +233,22 @@ export const REMOVE_DOMAIN = gql`
   }
 `
 
+export const REMOVE_USER_FROM_ORG = gql`
+  mutation RemoveUserFromOrg($userId: ID!, $orgId: ID!) {
+    removeUserFromOrg(input: { userId: $userId, orgId: $orgId }) {
+      result {
+        ... on AffiliationError {
+          code
+          description
+        }
+        ... on RemoveUserFromOrgResult {
+          status
```

Comment:
I believe you want to also add in a call to pull out the user name?
```graphql
mutation {
  removeUserFromOrg {
    result {
      ... on RemoveUserFromOrgResult {
        status
        user {
          userName
        }
      }
      ... on AffiliationError {
        code
        description
      }
    }
  }
}
```

Reply:
100% right, updated to grab name for onCompleted toast